scala-package/examples/src/main/scala/ml/dmlc/mxnetexamples/inferexample/SSDClassifierExample.scala

```diff
@@ -0,0 +1,96 @@
+package SSDClassifierExample
+
+import ml.dmlc.mxnet._
+import ml.dmlc.mxnet.infer._
+import org.kohsuke.args4j.{CmdLineParser, Option}
```

Comment:
Any comment why we need these things (next four lines)

Reply:
Used the ObjectDetection Class and ImageClassifier Object from mxnet.infer package. Do I need to import them separately in two lines or adding some comments in the code?

Will double check the functions being imported from mxnet._

Args4j used to get the user input args (such as '--model-dir'), same question here, do I need to add comments about this usage?